pkg/kubelet/images/types.go

```diff
@@ -19,7 +19,7 @@ package images
 import (
 	"errors"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Sorry I missed this one, looks like a change from `gopls` formatting the file on save, would you like me to remove?